src/frontend/src/lib/utils/tokens.utils.ts

```diff
@@ -80,3 +80,11 @@ export const pinTokensWithBalanceAtTop = ($tokens: TokenUi[]): TokenUi[] => {
 		...nonPositiveBalances
 	];
 };
+
+/**
+ * Calculates total USD balance of the provided tokens list.
+ *
+ * @param $tokens - The list of tokens for total USD balance calculation.
```

Comment:
If you add the documentation then maybe also document the return type?

Reply:
Done.